intakebuilder/gfdlcrawler.py

```diff
@@ -27,14 +28,17 @@ def crawlLocal(projectdir, dictFilter,dictFilterIgnore,logger):
         if(pat is not None):
             m = re.search(pat, searchpath)
             for filename in files:
+               if filename.startswith("."):
```

Comment:
need a test documented that you've tried out already.

Reply:
Line commented out for now.